provider/cmd/pulumi-resource-vsphere/schema.json

```diff
@@ -2901,6 +2902,7 @@
             }
         },
         "vsphere:index/computeClusterHostGroup:ComputeClusterHostGroup": {
+            "description": "The `vsphere.ComputeClusterHostGroup` resource can be used to manage groups\nof hosts in a cluster, either created by the\n`vsphere.ComputeCluster` resource or looked up\nby the `vsphere.ComputeCluster` data source.\n\n\nThis resource mainly serves as an input to the\n`vsphere.ComputeClusterVmHostRule`\nresource - see the documentation for that resource for further details on how\nto use host groups.\n\n\u003e **NOTE:** This resource requires vCenter and is not available on direct ESXi\nconnections.\n\n## Importing\n\nAn existing group can be [imported][docs-import] into this resource by\nsupplying both the path to the cluster, and the name of the host group. If the\nname or cluster is not found, or if the group is of a different type, an error\nwill be returned. An example is below:\n\n\n```\nterraform import vsphere_compute_cluster_host_group.cluster_host_group \\\n  '{\"compute_cluster_path\": \"/dc1/host/cluster1\", \\\n  \"name\": \"pulumi-test-cluster-host-group\"}'\n```\n",
```

Comment:
This includes a `terraform import` statement. Similar such in other descriptions.

Reply:
The bridge isn't picking them up for translation. Good catch.

Reply:
That was apparently a separate issue, now fixed.